src/Nethermind/Nethermind.JsonRpc/Data/TransactionForRpc.cs

```diff
@@ -39,8 +39,11 @@ public TransactionForRpc(Keccak blockHash, BigInteger? blockNumber, int? txIndex
             GasPrice = transaction.GasPrice;
             Gas = transaction.GasLimit;
             Input = Data = transaction.Data ?? transaction.Init;
+            R = transaction.Signature.R;
```

Comment:
There may be cases when we send unsigned transactions so I would suggest ?.R and ?.S and ?.V (elvis operator)

Reply:
ah yes of course. changed it accordingly